src/build/experimental/commands/compile/wasm_binary/rust/experimental_canister_template/src/execute_method_js.rs

```diff
@@ -13,7 +13,16 @@ pub extern "C" fn execute_method_js(function_index: i32, pass_arg_data: i32) {
 
         runtime.run_with_context(|context| {
             let global = context.get_global();
-            let callbacks = global.get("_azleCallbacks");
+
+            let exported_canister_class_instance = global.get("_azleExportedCanisterClassInstance");
```

Comment:
Is this for if you are using the class syntax in experimental? It's got to be right?